spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/ChannelBindingService.java

```diff
@@ -53,33 +32,16 @@
  * @author Marius Bogoevici
  * @author Ilayaperumal Gopinathan
  */
-public class ChannelBindingService implements InitializingBean {
+public class ChannelBindingService {
 
+	@Autowired
 	private Binder<MessageChannel> binder;
 
+	@Autowired
 	private ChannelBindingServiceProperties channelBindingServiceProperties;
 
-	private CompositeMessageConverterFactory messageConverterFactory;
-
-	public ChannelBindingService(ChannelBindingServiceProperties channelBindingServiceProperties, Binder<MessageChannel> binder) {
```

Comment:
Kind of a nit, but I think we should keep the `Binder` and `ChannelBindingServiceProperties` configured via constructor   to keep things clear (for that matter we could make them final too). The mix of no-arg construction in `ChannelBindingServiceConfiguration` and field autowiring makes this a little hard to follow, IMO.